config/locales/en/activerecord/errors/models/achievement_condition.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  activerecord:
+    errors:
+      models:
+        achievement_condition:
+          duplicate: 'achievement cannot have duplicate conditions'
```

Comment:
should be attribute.validation? See Rails Guides, i18n, section 4.6.1 Error Message Scopes